### ISSUE TITLE
Two problems with firefox45

### ISSUE THREAD

**Issue by kesdoputr** [vimperator/vimperator-labs#453]
Hello i'm using the newest vimperator 3.12.0,i just update to firefox 45.0 and find two problems.
01.The **restart command doesn't work**,when i type :res and press enter,firefox just close not restart.

02.When searching in web page,for example:

●press "/" to search in some web and type some words that don't match,like "tttt"
●press "n"(the hot key to find the next match pattern,but now there are no match pattern in this example),then the **firefox's search bar appear,**when i use 44.02 the search bar don't appear.
here is the example picture:
https://i.imgur.com/LGlaMp7.jpg
Or thers is a option can disable firefox's search bar?I have set those hotkeys to <Nop>that will call the bar like Ctrl+F F3 S+F3 etc...)because vimperator's search bar is enough.

**Comment by gkatsev** [comment]
Thanks for the detailed issue.
1. sounds like a bug that we have to fix
2. It's because vimperator is using the firefox findbar that it is happening, I think that the latest version 3.12.0 *should* fix it but maybe there's some other issues related to it, at least in firefox 45.

**Comment by kesdoputr** [comment]
Thanks for reply,problem 02 doesn't occur in firefox 44.02 with 3.12.0.When i press "n" on web page nothing occur just say pattern not found but on 45.0 will call the firefox's search bar.Maybe there's some different with 44.02 and 45.0.

**Comment by SammysHP** [comment]
```
Vimperator:	3.12.0 (created: 2016/03/02 13:30:53)
Firefox:	Mozilla/5.0 (X11; Linux x86_64; rv:45.0) Gecko/20100101 Firefox/45.0
```

Cannot reproduce the second issue.

**Comment by Jehan** [comment]
Just sneaking in to say that the second issue also occurs on Fedora 23:

> Vimperator:	3.12.0 (created: 2016/02/28 22:07:56)
> Firefox:	Mozilla/5.0 (X11; Fedora; Linux x86_64; rv:45.0) Gecko/20100101 Firefox/45.0

So that's not a Windows only problem. Linux too.
It recently appeared (probably when updating to Firefox 45, but I did not make accurate checks) and this is extremely annoying since not only it opens Firefox search bar, but it even gives focus in the search field, hence will start writing (erasing my previous search pattern) in it, if I click again 'n' or any other key.
Another annoying part is that I could not find any shortcut to close this search bar when it appears (ctrl-f opens it but does not close it back). So each time it comes up, it forces me to reach the mouse.

I would really welcome a fix, or if someone can find a temporary workaround at least, that would be cool. :-)

**Comment by SammysHP** [comment]
Actually this was fixed in #443. Sorry, I thought that it was included in 3.12.0.

**Comment by Jehan** [comment]
Oh ok. Well it's not. But it's cool that it's fixed. I'll just wait for the next release, no problem. :-)
Thanks.

**Comment by timss** [comment]
@Jehan If you don't feel like waiting, you could have Firefox use the latest version from the Git repo using this guide: https://github.com/vimperator/vimperator-labs/blob/master/CONTRIBUTING.md#development-installation

**Comment by ezyang** [comment]
Also affected by (2), would be lovely if there would be a release soon.

**Comment by timss** [comment]
Fix included in [3.13.0](https://github.com/vimperator/vimperator-labs/releases/tag/vimperator-3.13.0), please also see https://github.com/vimperator/vimperator-labs/issues/398#issuecomment-215306878.
